### ISSUE TITLE
Detected integration that called async_timeout.timeout with loop keyword argument.

### ISSUE THREAD

**Issue by Jefe2** [custom-components/weatheralerts#51]
**Version of the custom_component**
v 0.1.4

**Describe the bug**
After upgrading HA to 2021.12.0, a log entry states: “Detected integration that called async_timeout.timeout with loop keyword argument. …”

**Log**

Detected integration that called async_timeout.timeout with loop keyword argument. The loop keyword argument is deprecated and calls will fail after Home Assistant 2022.2. Please report issue to the custom component author for weatheralerts using this method at custom_components/weatheralerts/sensor.py, line 88: async with async_timeout.timeout(20, loop=hass.loop):
```
Add your logs here.
```

**Comment by ocean75** [comment]
Hello,
Receiving the same warning after updating to 2021.12.1.

**Log**
2021-12-13 07:49:28 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that called async_timeout.timeout with loop keyword argument. The loop keyword argument is deprecated and calls will fail after Home Assistant 2022.2. Please report issue to the custom component author for weatheralerts using this method at custom_components/weatheralerts/sensor.py, line 97: async with async_timeout.timeout(20, loop=hass.loop):
2021-12-13 07:49:28 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that called async_timeout.timeout with loop keyword argument. The loop keyword argument is deprecated and calls will fail after Home Assistant 2022.2. Please report issue to the custom component author for weatheralerts using this method at custom_components/weatheralerts/sensor.py, line 105: async with async_timeout.timeout(20, loop=hass.loop):
2021-12-13 07:49:28 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that called async_timeout.timeout with loop keyword argument. The loop keyword argument is deprecated and calls will fail after Home Assistant 2022.2. Please report issue to the custom component author for weatheralerts using this method at custom_components/weatheralerts/sensor.py, line 141: async with async_timeout.timeout(10, loop=self.hass.loop):
2021-12-13 07:49:28 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.weatheralerts_1 (<class 'custom_components.weatheralerts.sensor.WeatherAlertsSensor'>) implements device_state_attributes. Please report it to the custom component author.

**Comment by JOHLC** [comment]
Me as well

````
Detected integration that called async_timeout.timeout with loop keyword argument. The loop keyword argument is deprecated and calls will fail after Home Assistant 2022.2. Please report issue to the custom component author for weatheralerts using this method at custom_components/weatheralerts/sensor.py, line 88: async with async_timeout.timeout(20, loop=hass.loop):
Detected integration that called async_timeout.timeout with loop keyword argument. The loop keyword argument is deprecated and calls will fail after Home Assistant 2022.2. Please report issue to the custom component author for weatheralerts using this method at custom_components/weatheralerts/sensor.py, line 97: async with async_timeout.timeout(20, loop=hass.loop):
Detected integration that called async_timeout.timeout with loop keyword argument. The loop keyword argument is deprecated and calls will fail after Home Assistant 2022.2. Please report issue to the custom component author for weatheralerts using this method at custom_components/weatheralerts/sensor.py, line 105: async with async_timeout.timeout(20, loop=hass.loop):
Detected integration that called async_timeout.timeout with loop keyword argument. The loop keyword argument is deprecated and calls will fail after Home Assistant 2022.2. Please report issue to the custom component author for weatheralerts using this method at custom_components/weatheralerts/sensor.py, line 141: async with async_timeout.timeout(10, loop=self.hass.loop):
````

**Comment by jlverhagen** [comment]
Thanks for the reports! I'm working on this and will have an update ready soon.

**Comment by cbhiii** [comment]
Thanks @jlverhagen for your continued work on this very useful integration!

**Comment by jlverhagen** [comment]
This issue should now be resolved with v0.1.5.
